tests/test_jitterbuffer.py

```diff
@@ -114,14 +141,14 @@ def test_add_seq_too_high_discard_four(self):
         jbuffer.add(RtpPacket(sequence_number=1, timestamp=1234))
         self.assertEqual(jbuffer._origin, 0)
 
-        jbuffer.add(RtpPacket(sequence_number=2, timestamp=1234))
-        self.assertEqual(jbuffer._origin, 0)
+        #  jbuffer.add(RtpPacket(sequence_number=2, timestamp=1234))
```

Comment:
Please remove the code instead of commenting it.

Reply:
No problem I'll remove it - I just thought it's easier to understand what's going on this way.